### PR TITLE
add shift to eda

### PR DESCRIPTION
**Pull request by jsharpna**
*Description of changes:*

We have added the shift package to eda.  It has already gone through two round of review in other PRs.  The fundamental components of shift have been isolated for this merge to master.

### CODE REVIEW

**Comment by gradientsky** [comment]
This PR needs this in `setup.py`
```python
install_requires = [
    ...
    f'autogluon.features=={version}',
]
```

File: eda/src/autogluon/eda/analysis/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import phik
 from .base import Namespace
 from .dataset import Sampler
+from .shift import XShiftDetector
```
Comment (gradientsky) [inline eda/src/autogluon/eda/analysis/__init__.py:4]: nit: extra line at the end of file. Please run `flake8` in this package - it should highlight issues like this.

File: eda/src/autogluon/eda/visualization/__init__.py [hunk 2]
```diff
@@ -1 +1,2 @@
 from .layouts import MarkdownSectionComponent, SimpleVerticalLinearLayout, SimpleHorizontalLayout, TabLayout
+from .shift import *
```
Comment (gradientsky) [inline eda/src/autogluon/eda/visualization/__init__.py:2]: nit: extra line

File: eda/src/autogluon/eda/visualization/jupyter.py [hunk 3]
```diff
@@ -22,3 +22,40 @@ def fix_tabs_scrolling(self):
...
+                    border-radius: unset !important;
+                    -webkit-box-shadow: unset !important;
+                    box-shadow: unset !important;
+                }
+                .jupyter-widgets-output-area  {
+                    height: auto !important;
+                }
+            </style>
+            """
+        display(HTML(style))
```
Comment (gradientsky) [inline eda/src/autogluon/eda/visualization/jupyter.py:61]: This is not needed - it's in the class above

**Comment by jsharpna** [comment]
I also added the following in `setup.py`

```
    f'autogluon.core=={version}',
    f'autogluon.features=={version}',
    f'autogluon.tabular=={version}',
```

**Review by gradientsky — APPROVED** [review]

**Pull request merged**